Tribler/community/tunnel/tunnel_community.py

```diff
@@ -889,22 +883,27 @@ def on_cell(self, messages):
             if not self.relay_cell(circuit_id, message.payload.message_type, message):
 
                 plaintext, encrypted = TunnelConversion.split_encrypted_packet(message.packet, message.name)
+
+                # We don't know this circuit, so continue.
```

Comment:
How so? In the old crypto_in call the circuit was retrieved from self.circuits using the circuit_id. So wouldn't that fail then?
Since I am passing the circuit now instead of the id I figured to move the check which I is now removed below, up a bit. Where does my logic go wrong?